src/ext/java/org/opentripplanner/ext/transmodelapi/model/timetable/DatedServiceJourneyType.java

```diff
@@ -0,0 +1,56 @@
+package org.opentripplanner.ext.transmodelapi.model.timetable;
+
+import static org.opentripplanner.ext.transmodelapi.model.EnumTypes.SERVICE_ALTERATION;
+
+import graphql.Scalars;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLNonNull;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLOutputType;
+import graphql.schema.GraphQLTypeReference;
+import org.opentripplanner.ext.transmodelapi.support.GqlUtil;
+import org.opentripplanner.model.TripOnServiceDate;
+
+/**
+ * A DatedServiceJourney GraphQL Type for use in endpoints fetching DatedServiceJourney data
+ */
+public class DatedServiceJourneyType {
+
+    private static final String NAME = "DatedServiceJourney";
+    public static final GraphQLTypeReference REF = new GraphQLTypeReference(NAME);
+
+    public static GraphQLObjectType create(GraphQLOutputType serviceJourneyType) {
+        return GraphQLObjectType.newObject()
+                .name(NAME)
+                .description("A planned journey on a specific day")
+                .field(GqlUtil.newTransitIdField())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("OperatingDayDate")
```

Comment:
I think this should be just "operatingDay", it is not necessary to repeat the type. Also, make sure all fields start with lower case.